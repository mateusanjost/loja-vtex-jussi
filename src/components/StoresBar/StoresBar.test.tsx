import TestRenderer from "react-test-renderer";
import StoresBarComponent from "./StoresBar";

describe("Stores Bar", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<StoresBarComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
