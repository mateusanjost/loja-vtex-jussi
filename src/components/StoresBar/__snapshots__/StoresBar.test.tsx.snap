// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stores Bar should match snapshot 1`] = `
<div
  className="sc-bdnxRM eJTjuE"
>
  <div
    className="sc-gtsrHT hplHCM"
  >
    <strong>
      Nossas principais lojas vtex
    </strong>
    <b>
      â†’
    </b>
  </div>
  <div
    className="sc-dlnjwi lkunRg"
  >
    <a
      href="https://www.brastemp.com.br/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Brastemp Logo"
        src="logo-brastemp1x.svg"
        srcSet="logo-brastemp1x.svg, logo-brastemp2x.svg 2x, logo-brastemp3x.svg 3x"
      />
    </a>
    <a
      href="https://www.compracerta.com.br/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Compra certa Consul Logo"
        src="logo-comprafacil-consul1x.svg"
        srcSet="logo-comprafacil-consul1x.svg, logo-comprafacil-consul2x.svg 2x, logo-comprafacil-consul3x.svg 3x"
      />
    </a>
    <a
      href="https://loja.consul.com.br/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Consul Logo"
        src="logo-consul1x.svg"
        srcSet="logo-consul1x.svg, logo-consul2x.svg 2x, logo-consul3x.svg 3x"
      />
    </a>
    <a
      href="https://br.thebar.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="The bar Logo"
        src="logo-thebar1x.svg"
        srcSet="logo-thebar1x.svg, logo-thebar2x.svg 2x, logo-thebar3x.svg 3x"
      />
    </a>
  </div>
</div>
`;
